.gitignore

```diff
@@ -5,3 +5,6 @@ __pycache__
 .mypy_cache
 .venv
 .idea
+
+.DS_Store
+.vscode/launch.json
```

Comment:
Normally I like to check in and share `.vscode/launch.json` since it can help other to set up debugging easily. It doesn't matter if it might not work out of box -- it is merely a helpful info.